docs/model_library/operational_water_management/index.rst

```diff
@@ -575,15 +574,12 @@ Flow balance constraint (i.e., inputs are equal to outputs). For each pipeline n
 
 .. math::
 
-    \sum\nolimits_{(p,n)\in PNA}\textcolor{red}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(p,n)\in CNA}\textcolor{red}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(\tilde{n},n)\in NNA}\textcolor{red}{F_{l,l,t}^{Piped}}+\sum\nolimits_{(s,n)\in SNA}\textcolor{red}{F_{l,l,t}^{Piped}}
-
-        = \sum\nolimits_{(n,\tilde{n})\in NNA}\textcolor{red}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(n,p)\in NCA}\textcolor{red}{F_{l,l,t}^{Piped}}+\sum\nolimits_{(n,k)\in NKA}\textcolor{red}{F_{l,l,t}^{Piped}}
-
-        +\sum\nolimits_{(n,r)\in NRA}\textcolor{red}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(n,s)\in NSA}\textcolor{red}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(n,o)\in NOA}\textcolor{red}{F_{l,l,t}^{Piped}}
+     \sum_{l \in L | (l, n) \in LLA}\textcolor{red}{F_{l,n,t}^{Piped}}
+            = \sum_{l \in L | (n, l) \in LLA}\textcolor{red}{F_{n,l,t}^{Piped}}
 
 
 
-**Bi-Directional Flow:** :math:`\forall (l,l) \in {PCA,PNA,PPA,CNA,NNA,NCA,NKA,NSA,NRA,\ldots ,SOA}, t \in T`
+**Bi-Directional Flow:** :math:`\forall (l,\tilde{l}) if (l,\tilde{l}) \in LLA and (\tilde{l}, l) \in LLA , t \in T`
```

Comment:
The code shows that these constraints only apply to l in L - F - O and ltilde in L - F, so I think the documentation should reflect that.

Also, the way this is rendered doesn't look very good: 
![image](https://github.com/project-pareto/project-pareto/assets/26798258/1203b540-958e-49dd-9778-983bb8637405)

Multiple `:math:` tags should be used so that the words "and" and "if" aren't rendered in math mode.

Reply:
Addressed.